src/app/routes/routes.jsx

```diff
@@ -8,6 +8,7 @@ const routes = {
   default: (
     <Route path="/" component={Application}>
       <Route path="/search" component={Application} />
+      <Route path="/work" component={Application} />
```

Comment:
When you click on a work's title, you do not go to that route even though the component loads. What you want to do is update the component here to be `WorkDetail` to render that. Also, the `/search` path should load the `SearchResults` component. This would be tied down together with makes changes in the `Application` component so that you can render the children... something like
```
   <Application>
     <SearchContainer>
       <SearchForm>  //I am assuming you always want to render this component
         // either render
         <SearchResults> // for the /search path
         <WorkDetail> // for the /work path
```

Let's go over that next week? For now and for this PR, the rendering of different components is working and fine.

Reply:
Agreed that this needs to change in the routing. Am also having difficulty with server-side rendering with the Redux store and associated actions. Let's discuss next week for sure. 😄